model/mainframe/registry_lpar.yaml

```diff
@@ -0,0 +1,11 @@
+groups:
+  - id: registry.mainframe.lpar
```

Comment:
nit:

No strong opinion on this one, but is there any reason to limit this group (and file) to lpar namespace? Registry groups have no special meaning - they are just collections of attributes, so we usually put all the attributes that share one namespace to the same registry group.

Reply:
Comment from mainframe SIG: this should probably be kept to 'lpar'. Everything within a mainframe runs within an LPAR, and we don't have a good alternative if we wanted to remove this.